spec/functional/combined.spec.js

```diff
@@ -34,7 +34,7 @@ describe('combined inbox/outbox flows', function () {
     return global.resetDb(apex, client, testUser)
   })
 
-  it('adds followers and delivers to them', async function (done) {
+  it('adds followers and delivers to them', async function () {
```

Comment:
This one has an undefined `done` and it going to need that clever promise construction pattern you used with similar tests in outbox.spec.js

Reply:
Yes, I'll add that. Thanks again.